language/stdlib/modules/vector.mvir

```diff
@@ -5,9 +5,17 @@ module Vector {
   // Vector containing data of type Item
   native struct T<Element>;
 
+  native public empty<Element>(): Self.T<Element>;
```

Comment:
I prefer `empty`, but I will bring it up regardless: Thoughts on `new`?

Reply:
I would be fine with `new`, but also prefer `empty`.